Makefile

```diff
@@ -31,7 +31,7 @@ clean:
 	rm -rf dist/*
 
 test:
-	go test -cover -v `glide novendor`
+	sh go.test.sh
```

Comment:
This seems to complicated for me. Can we not just run `go test -race -coverprofile=profile.out -covermode=atomic ./poll/`